tjreports/site/views/reports/view.csv.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
@tjbhavesh check if we are using this file otherwise remove this from front and backend